### ISSUE TITLE
i download  phreebook erp_master from github how to implement it and syn with opencart website 

### ISSUE THREAD

**Issue by varsha63** [phreebooks/PhreeBooksERP#216]
pls help me i am working on opencart project and want to interface phreebook erp with opencart admin panel and all the records can you pls suggest any module or any it will be very help full

**Comment by alvin101** [comment]
http://www.opencart.com/index.php?route=extension/extension/info&extension_id=6267&filter_search=phreebooks

https://www.opencart.com/index.php?route=extension/purchase&extension_id=6267&filter_search=phreebooks

**Comment by phreesoft** [comment]
Did you buy the OpenCart Interface? It is available on the OpenCart site 
and the PhreeBooks site. There are instructions included with the extension.

Dave

**Comment by varsha63** [comment]
yes i buy the extension today now i have trunk folder of phresoft erp and opencart folders admin and vqmod i done half of the installation i am not  getting inventory/ edit-maintain option in phreebook erp what to do and tell me what is the use of extensions folder comes with phreebook erp which is download from github phreebooks .

**Comment by phreesoft** [comment]
You need to install PhreeBooks first and then install the extension. There are installation instructions in the trunk folder. Once your server is set up and both PhreeBooks and OpenCart are up and running you can then install the Interface. There are instructions for the interface with the package. 

Setting up your server and installation of the core PhreeBooks and OpenCart are beyond my support for the interface. You can also get support on the PhreBooks forum and OpenCart forum..

Dave

**Comment by varsha63** [comment]
how to get products list from opencart to phreebook erp inventory  

its not fetching customer and vendors lists

**Comment by phreesoft** [comment]
The interface only upload products from PhreeBooks to OpenCart (same with all my other interfaces). Orders (which include the customer information for the order) are only downloaded to PhreeBooks. 

Dave

**Comment by varsha63** [comment]
and if i export product list as csv or xml then i can import that file to inventory is it work

**Comment by varsha63** [comment]
and i having one more issue once i make user and give full permission to that user but after login in today i don't able to edit new general ledger where add symbol not working and save button is also not working of all the company and in users also not editable i download phreebook from github and interface from opencart any issue regarding both compatible issue or any other step i am missing that why i am not able to save , edit everything in phreebook erp its coming second time. 
pls ans asap
